packages/e2e-tests/src/tests/WrappedKeys.spec.ts

```diff
@@ -0,0 +1,1211 @@
+      bs58.decode(generatedPublicKey)
+    );
+
+    if (!signatureIsValidForPublicKey)
+      throw new Error(
+        `signature: ${signature} doesn't validate for the Solana public key: ${generatedPublicKey}`
+      );
+
+    const pkpSessionSigsExport = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    const { decryptedPrivateKey } = await exportPrivateKey({
+      pkpSessionSigs: pkpSessionSigsExport!,
+      litNodeClient: devEnv?.litNodeClient!,
+      network: 'solana',
+      id,
+    });
+
+    const solanaKeyPair = Keypair.fromSecretKey(
+      Buffer.from(decryptedPrivateKey, 'hex')
+    );
+    const decryptedPublicKey = solanaKeyPair.publicKey;
+
+    expect(decryptedPublicKey.toString()).toEqual(generatedPublicKey);
+  });
+
+  it('Generate ETH Wrapped Key', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    const pkpSessionSigs = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    const { pkpAddress, generatedPublicKey, id } = await generatePrivateKey({
+      pkpSessionSigs: pkpSessionSigs!,
+      network: 'evm',
+      litNodeClient: devEnv.litNodeClient!,
+      memo: 'Test Key',
+    });
+
+    const alicePkpAddress = alice.authMethodOwnedPkp?.ethAddress;
+    if (pkpAddress !== alicePkpAddress) {
+      throw new Error(
+        `Received address: ${pkpAddress} doesn't match Alice's PKP address: ${alicePkpAddress}`
+      );
+    }
+
+    const pkpSessionSigsExport = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    const { decryptedPrivateKey } = await exportPrivateKey({
+      pkpSessionSigs: pkpSessionSigsExport!,
+      litNodeClient: devEnv.litNodeClient!,
+      network: 'evm',
+      id,
+    });
+
+    const wallet = new ethers.Wallet(decryptedPrivateKey);
+    const decryptedPublicKey = wallet.publicKey;
+
+    expect(decryptedPublicKey).toEqual(generatedPublicKey);
+  });
+
+  it('Fail Import Wrapped Key Same Private Key', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    const pkpSessionSigs = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    const privateKey =
+      '4rXcTBAZVypFRGGER4TwSuGGxMvmRwvYA3jwuZfDY4YKX4VEbuUaPCWrZGSxujKknQCdN8UD9wMW8XYmT1BiLxmB'; // Already exists in the DB
+
+    try {
+      await importPrivateKey({
+        pkpSessionSigs: pkpSessionSigs!,
+        privateKey,
+        litNodeClient: devEnv.litNodeClient!,
+        publicKey: '0xdeadbeef',
+        keyType: 'K256',
+        memo: 'Test Key',
+      });
+    } catch (e: any) {
+      if (
+        e.message.includes(
```

Comment:
We can use expect.toThrow() here instead, and :hocho: the extra `console.log`
See also https://jestjs.io/docs/expect#rejects :)